docs/osf-trolling-list/build_process.md

```diff
@@ -0,0 +1,127 @@
+`[OSFI001]` Build process
+-------------------------
+
+# `[OSFI0010]` There is no healthy OSF without reproducible builds
+
+## What are the reproducible builds?
+
+[Reproducible Builds](https://reproducible-builds.org/) and lot of its aspects
+is well explained on linked website.
+
+## Why those are in particular important for OSF?
+
+This is because knowing hashes of firmware components we can calculate expected
```

Comment:
Please make sure to spellcheck this all at the end _(Grammarly etc. Grammarly is good, although imperfect)_

Reply:
@pietrushnic ofc it's fine to share a not-spellchecked text for a preliminary review _(based on feedback the text could change and time spellchecking it would be lost)_ - just a small warning that a reviewer could ignore "the typos" would be nice.

Reply:
@mikebdp2 I see that @KamilRadzewicz already pushed spelling fixes, please resolve it it looks good for you.